database_email_backend/admin.py

```diff
@@ -43,7 +43,7 @@ def file_link(self, obj):
 
 
 class EmailAdmin(admin.ModelAdmin):
-    list_display = ('from_email', 'to_emails', 'subject', 'body_stripped', 'sent_at', 'attachment_count')
+    list_display = ('from_email','reply_to', 'to_emails', 'subject', 'body_stripped', 'sent_at', 'attachment_count')
```

Comment:
I will fix the reply_to in one commit. I will do the pep8 lovin' in a seperate commit